governance/ChaosDAOInternalProposals.md

```diff
@@ -0,0 +1,16 @@
+# Internal ChaosDAO Governance Procedures
+
+## How to Propose?
+Generally, proposals often need some initial discussion to hammer specific proposal parameters. In those cases, DAOists should first bring up proposal topics in a new #daist-forum discussion thread. Some quick single-parameter votes can be held, even including multiple options for people to vote between. That provides insight into what exactly the proposal should include.
```

Comment:
I would switch hammer to "provide"

Reply:
I would also swap out the phrase "bring up" to "introduce"

Reply:
`initial discussion to hammer specific proposal parameters`
changed to:
`initial discussion to **clarify** specific proposal parameters`